### ISSUE TITLE
Add support for excludeFiles to remove warnings?

### ISSUE THREAD

**Issue by jliebrand** [AnomalyInnovations/serverless-bundle#97]
I have my handler code like this:
```
/src
  foo.js
  foo.spec.js
```

I'm getting warnings about finding multiple handlers, which stems from this:
https://github.com/serverless-heaven/serverless-webpack/issues/433

Not sure the best way to add support for excludeFiles in serverless-bundle though? I.e. I can pipe it through and create a PR, but is that what we want or do we want the bundle to do some sensible defaults here instead?

**Comment by jliebrand** [comment]
e.g. Convention over configuration ?

**Comment by jayair** [comment]
Yeah might be a good idea to use a default `excludeFiles` option and then allow people to override or extend it?

**Comment by jliebrand** [comment]
Sorry - I think I missed the notification of your comment. Yes, I'd say a default exclusion of "*.spec.js" and "*.test.js" would go a long way

**Comment by RobertGardner** [comment]
> Sorry - I think I missed the notification of your comment. Yes, I'd say a default exclusion of "_.spec.js" and "_.test.js" would go a long way

and "_.spec.ts_" and "_.test.ts_"

**Comment by jayair** [comment]
Sounds good. If somebody wants to add this option with these defaults and open a PR that would be great!
